.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,34 @@
+[//]: # (Title Template: "[TR_##] Title")
+
+---
+
+#### Please check if the PR fulfills these requirements
+
+- [ ] The changes don't come from your master branch.
```

Comment:
Should we add a link to a 'rebasing best-practices'?
Otherwise, looks great!

Reply:
that would be good! idk what else should be in this list of requirements though, and if the current list itemshould even be in it? idk